.github/scripts/compile.sh

```diff
@@ -133,7 +133,10 @@ do_build () {
 
 # Do a regular build without asan or tsan.
 do_build "" OFF
+# and install Proton Python
+python3 -m pip install --prefix="$PROTON_INSTALL_DIR/usr" "$(find "$PROTON_BUILD_DIR/python/" -name 'python-qpid-proton-*.tar.gz')"
```

Comment:
@astitcher Is there a reason why the wheel is named python_qpid_proton and tar.gz is python-qpid-proton? It is a bit weird.

Reply:
It seems that is just what the tooling does - I don't knwo why.